www/install/steps/process/createDbUser.php

```diff
@@ -63,7 +63,7 @@
 $host = "localhost";
 // if database server is not on localhost...
 if ($parameters['address'] != "127.0.0.1" && $parameters['address'] != "localhost") {
-    $host = $_SERVER['SERVER_ADDR'];
+    $host = explode(" ", $link->getAttribute(PDO::ATTR_CONNECTION_STATUS))[0];
```

Comment:
The value returned by *PDO::ATTR_CONNECTION_STATUS* seems to be undefined to say the least.

What do you think about reworking our hostname retrieval with a **SELECT USER()** query ? This function is defined for both [MySQL](https://dev.mysql.com/doc/refman/8.0/en/information-functions.html#function_user) and [MariaDB](https://mariadb.com/kb/en/library/user/).

Reply:
Yes @ganoze I was verifying exactly that, it was much more effective I directly query mysql. I ended up doing a little different from what he suggested, but in the same way.
Thanks